tests/integration_tests/datasources/test_lxd_hotplug.py

```diff
@@ -56,8 +56,9 @@ def _prefer_lxd_datasource_over_nocloud(client: IntegrationInstance):
     client.restart()
 
 
+# TODO: Once LXD adds MACs to the devices endpoint, support LXD VMs here
+# Currently the names are too unpredictable to be worth testing on VMs.
 @pytest.mark.lxd_container
-@pytest.mark.lxd_vm
```

Comment:
This is fine as-is as we are ultimately only testing that new network-config is applied wholesale across hot-plug device boundaries. So, this test coverage won't make a difference VM vs container, because the network-config author will have to know beforehand that their VMs happen to name devices enp5s0 enp6s0 instead of eth0 eth1 etc. network-config can be flexible in those cases and set globbing if needed `enp*:`  vs `eth*:`.